cargo-generate.toml

```diff
@@ -1,10 +1,13 @@
 [template]
 cargo_generate_version = "0.21.3"
 ignore = [
+    ".github/workflows/ci-generate.yaml",
```

Comment:
```suggestion
```
Optional. We can add this line in the PR that adds this workflow, but I'm fine either way.